internal/testing/e2e/backfill_test.go

```diff
@@ -238,6 +206,26 @@ func TestProposedBackfillUpdate(t *testing.T) {
 			Score: 10,
 		}),
 	}
+	createMatchWithBackfill(ctx, om, b, t)
+	client, err := om.Query().QueryTickets(ctx, &pb.QueryTicketsRequest{Pool: &pb.Pool{
+		StringEqualsFilters: []*pb.StringEqualsFilter{{StringArg: "field", Value: "value"}},
+	}})
+
+	require.Nil(t, err)
```

Comment:
```suggestion
	require.NoError(t, err)

```

Reply:
I was testing a task on VS Code. And this is a leftover. changed back already.

Reply:
good catch. 
@aLekSer please update it everywhere in all test files you are working with